server/update.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This scrip pulls and builds the latest version of the site.
+# It can be run manually, but it is much more useful as a cron job.
+#
+# For example, this crontab entry will build the site every 15 minutes.
+#
+#   0 */15 * ? * * sh /home/admin/delta.khk.org/server/update.sh /home/admin/delta.khk.org /var/www/delta.khk.org
+
+if [[ $# -eq 0 ]] ; then
```

Comment:
Should there be exactly one argument? If so, a better check might be:
```bash
if [[ $# != 1 ]]; then
```

Reply:
There should be exactly two arguments. I'm not super familiar with bash. What would be a good way to check this?

Reply:
You could do something like:
```bash
if [[ 2 != $# ]]; then
    printf '\n%s\n' "[$(date)]Jekyll Build Failed: No path to Jekyll site supplied. Please provide two arguments. \n"
fi
```